distribution/install-profiles/src/main/resources/features.xml

```diff
@@ -29,31 +29,9 @@
         <feature>catalog-solr-app</feature>
 
         <!-- Alliance Apps -->
+        <feature>video-app</feature>
```

Comment:
❓ Intersting so video-app was only in the profile-alliance-experimental profile?

Reply:
Yes other than resource mgmt and GWC.  Video app should have been moved out of experimental long ago given its  community use.